ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -220,6 +220,9 @@ router bgp {{ router_bgp.as }}
 {%         if neighbor.description is arista.avd.defined %}
    neighbor {{ neighbor.ip_address }} description {{ neighbor.description }}
 {%         endif %}
+{%         if neighbor.route_reflector_client is arista.avd.defined(true) %}
```

Comment:
For this parameter we want to render the negative case too, since it is common to see a peer group with route_reflector_client set but then disabling for a single peer.

Reply:
Fixed, thanks!